core/src/main/java/bio/terra/pearl/core/service/export/formatters/item/AnswerItemFormatter.java

```diff
@@ -90,9 +111,25 @@ public AnswerItemFormatter(ExportOptions exportOptions, String moduleName, Surve
         questionType = questionDef.getQuestionType();
         questionText = questionDef.getQuestionText();
         hasOtherDescription = questionDef.isAllowOtherDescription();
+        isChildQuestion = questionDef.getParentStableId() != null;
+        parentStableId = questionDef.getParentStableId();
         versionMap = new HashMap<>();
+        this.objectMapper = objectMapper;
     }
 
+    private static String buildBaseKey(String questionStableId, Integer repeatIndex, String parentStableId) {
+        String key = questionStableId;
+        if (Objects.nonNull(repeatIndex)) {
+            key += "[" + repeatIndex + "]";
```

Comment:
create a helper for this in ExportFormatUtils. something like 'formatIndex' that takes an int and returns "[%s]".formatted(int)